plugins/services/src/js/constants/ServiceStatusTypes.ts

```diff
@@ -0,0 +1,12 @@
+enum ServiceStatusTypes {
+  NA = 0,
+  STOPPED = 1,
+  RUNNING = 2,
+  DEPLOYING = 3,
+  WAITING = 4,
+  DELAYED = 7,
+  RECOVERING = 5,
+  DELETING = 6
```

Comment:
Could we use this with `ServiceStatusInterface` & `ServiceStatusLabelsInterface`?

Reply:
I think we can, but refactoring the constants is for me out of scope here.